tinker/events/__init__.py

```diff
@@ -47,6 +47,8 @@ def index(self):
                                UserMatches=None, matches=None)
         user_forms = self.base.traverse_xml(app.config['EVENTS_XML_URL'], 'system-page')
         if 'Tinker Events - CAS' in session['groups'] or 'Event Approver' in session['groups']:
+            user_forms = self.base.traverse_xml(app.config['EVENTS_XML_URL'], 'system-page')
```

Comment:
`if 'Tinker Events - CAS' in session['groups'] or 'Event Approver' in session['groups']:`

This line looks to be duplicated? maybe an issue with the update from master conflicts?